app/src/main/java/io/gnosis/safe/ui/transaction/TransactionViewHolderFactory.kt

```diff
@@ -36,87 +56,223 @@ class TransactionViewHolderFactory : BaseFactory<BaseTransactionViewHolder<Trans
+
+            finalStatus.alpha = OPACITY_FULL
+            amount.alpha = viewTransfer.alpha
+            dateTime.alpha = viewTransfer.alpha
+            txTypeIcon.alpha = viewTransfer.alpha
+            blockies.alpha = viewTransfer.alpha
+            ellipsizedAddress.alpha = viewTransfer.alpha
+        }
+    }
+}
+
+class TransferQueuedViewHolder(private val viewBinding: ItemTxQueuedTransferBinding) :
+    BaseTransactionViewHolder<TransactionView.TransferQueued>(viewBinding) {
+    private val resources = viewBinding.root.context.resources
+
+    override fun bind(viewTransfer: TransactionView.TransferQueued, payloads: List<Any>) {
+        with(viewBinding) {
+            status.setText(viewTransfer.statusText)
+            status.setTextColor(resources.getColor(viewTransfer.statusColorRes))
+            amount.text = viewTransfer.amountText
+            dateTime.text = viewTransfer.dateTimeText
+            txTypeIcon.setImageResource(viewTransfer.txTypeIcon)
+            blockies.setAddress(viewTransfer.address)
+            ellipsizedAddress.text = viewTransfer.address.formatForTxList()
+            amount.setTextColor(resources.getColor(viewTransfer.amountColor))
+            confirmations.setTextColor(resources.getColor(viewTransfer.confirmationsTextColor))
+            confirmationsIcon.setImageDrawable(resources.getDrawable(viewTransfer.confirmationsIcon))
+            confirmations.text = resources.getString(R.string.tx_list_confirmations, viewTransfer.confirmations, viewTransfer.threshold)
+            nonce.text = viewTransfer.nonce
+        }
     }
 }
 
 class SettingsChangeViewHolder(private val viewBinding: ItemTxSettingsChangeBinding) :
     BaseTransactionViewHolder<TransactionView.SettingsChange>(viewBinding) {
+    private val resources = viewBinding.root.context.resources
 
-    override fun bind(data: TransactionView.SettingsChange, payloads: List<Any>) {
+    override fun bind(viewTransfer: TransactionView.SettingsChange, payloads: List<Any>) {
         with(viewBinding) {
-            settingName.text = data.transaction.dataDecoded.method
-            dateTime.text = data.transaction.date
+            finalStatus.text = viewTransfer.status.name
+            finalStatus.setTextColor(statusTextColor(viewTransfer.status, resources))
+            dateTime.text = viewTransfer.dateTimeText
+            settingName.text = viewTransfer.settingNameText
+
+            finalStatus.alpha = OPACITY_FULL
+            dateTime.alpha = viewTransfer.alpha
+            settingName.alpha = viewTransfer.alpha
         }
     }
 }
 
-class TransferViewHolder(private val viewBinding: ItemTxTransferBinding) :
-    BaseTransactionViewHolder<TransactionView.Transfer>(viewBinding) {
+class SettingsChangeQueuedViewHolder(private val viewBinding: ItemTxQueuedSettingsChangeBinding) :
+    BaseTransactionViewHolder<TransactionView.SettingsChangeQueued>(viewBinding) {
+    private val resources = viewBinding.root.context.resources
```

Comment:
Same, here. Keeping a reference to resource that can be accessed when it's required by `bind`